src/utils/kycEncryption.ts

```
@@ -0,0 +1,41 @@
+  } catch {
+    return null;
+  }
+}
+
+export function encryptKycPayload(payload: unknown): string | null {
+  if (payload == null) return null;
+  const key = getKey();
+  if (!key) {
+    throw new Error(
+      "PII_ENCRYPTION_KEY not configured — refusing to store plaintext KYC payload",
+    );
```

ESLint

[error] Replace `⏎······"PII_ENCRYPTION_KEY·not·configured·—·refusing·to·store·plaintext·KYC·payload",⏎····` with `"PII_ENCRYPTION_KEY·not·configured·—·refusing·to·store·plaintext·KYC·payload"`

(prettier/prettier)